kfs-core/src/main/resources/edu/arizona/kfs/fp/document/workflow/AuxiliaryVoucherDocument.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version='1.0' encoding='UTF-8'?> 
+<data xmlns="ns:workflow" xmlns:fo="http://www.w3.org/1999/XSL/Format" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="ns:workflow resource:WorkflowData">
+  <documentTypes xmlns="ns:workflow/DocumentType" xsi:schemaLocation="ns:workflow/DocumentType resource:DocumentType">
+    <documentType>
+      <name>AV</name>
+      <parent>FP</parent>
+      <label>Auxiliary Voucher</label>
+      <docHandler>${application.url}/financialAuxiliaryVoucher.do?methodToCall=docHandler</docHandler>
+      <routePaths>
+        <routePath>
+          <start name="AdHoc" nextNode="Account"/>
+          <requests name="Account" nextNode="ObjectSubTypeCode"/>
```

Comment:
Seems to be missing:
 role name="Account" nextNode="AccountingOrganizationHierarchy"
role name="AccountingOrganizationHierarchy" nextNode="ObjectSubTypeCode"
It seems like this file was copied from KFS 3.0 code rather than KFS 6.0 code:
kfs-core/src/main/config/workflow/003_kfs_core_module_docs/FinancialProcessingTransactionalDocuments.xml